app/js/states/digests/digests.controller.js

```diff
@@ -144,6 +146,40 @@ function DigestsCtrl($scope, DigestService, System, Rule, InventoryService, Seve
             };
         }
 
+        $scope.downloading = false;
+        $scope.downloadPdf = function () {
+            const digestId = res.data.resources[0].digest_id;
+            const uri = URI(InsightsConfig.apiRoot)
+                .segment('digests')
+                .segment(digestId.toString())
+                .toString();
+
+            $scope.downloading = true;
+            $http.get(uri, {
+                responseType: 'arraybuffer',
+                headers: { Accept: 'application/pdf'}
+            })
```

Comment:
We should handle errors as well in case something goes wrong. We probably want to show an error message of some sort but at the very least we should set $scope.downloading to false.